tox.ini

```diff
@@ -117,10 +123,12 @@ commands_pre=
     mkdir -p test-results .coverage
     core: ./wini package wandb-core --coverage --install
 commands=
-    pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 {posargs}
+    importer-system-wandb: pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 --wandb-second-server="true" {posargs}
+    !system: pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 {posargs}
```

Comment:
you probably don't need these when you run locally just add an extra flag: 
tox -e .. -- --wandb-second-server...